test/test_syntax.tt

```diff
@@ -1921,3 +1921,7 @@ monitor(@com.twitter.home_timeline() filter any(sort(file_size asc of @com.dropb
 monitor(@com.twitter.home_timeline() filter any(@com.twitter.post()));
 
 
+====
+
+// comparison subquery infix comparison operator
+@org.wikidata.city() filter in_array(id, any([shares_border_with] of @org.wikidata.city() filter id =~ "palo alto"));
```

Comment:
Uh? This test case is not infix?

Reply:
Ugh I mean postfix for this whole PR... I will update this once I got back

Reply:
You mean prefix...

Reply:
> You mean prefix...

yes...